gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/RequestHandler.java

```diff
@@ -36,6 +36,7 @@
 
 public class RequestHandler implements RequestHandlerWithSorting {
   private static final Logger LOG = LoggerFactory.getLogger(RequestHandler.class);
+
```

Comment:
Is this intentional?

Reply:
yes, I prefer to keep a gap between static and non static class variables